CI/package-win.sh

```
@@ -102,21 +102,21 @@ while IFS= read -r line || [[ -n "$line" ]]; do
-    cp "${GITHUB_WORKSPACE_UNIX_PATH}/build-${gameName}/MudletBootstrap.exe.debug" "${PACKAGE_DIR}/"
+  cp "${GITHUB_WORKSPACE_UNIX_PATH}/build-${gameName}/MudletInstaller.exe" "${PACKAGE_DIR}/"
+  if [ -f "${GITHUB_WORKSPACE_UNIX_PATH}/build-${gameName}/MudletInstaller.exe.debug" ]; then
+    cp "${GITHUB_WORKSPACE_UNIX_PATH}/build-${gameName}/MudletInstaller.exe.debug" "${PACKAGE_DIR}/"
   fi
 
 
-  "${RUNNER_WORKSPACE}/qt-static-install/bin/windeployqt6" ./MudletBootstrap.exe
+  "${RUNNER_WORKSPACE}/qt-static-install/bin/windeployqt6" ./MudletInstaller.exe
 
-  ZIP_FILE_NAME="MudletBootstrap"
+  ZIP_FILE_NAME="MudletInstaller"
```

Shellcheck (0.11.0)

[warning] ZIP_FILE_NAME appears unused. Verify use (or export if used externally).

(SC2034)